README.md

```diff
@@ -1,63 +1 @@

OUR PATICIPATION IN GOOGLE CODE-IN.

Every year we participate in the competition google code in organized by google. This competition is for pre-university (13 -14 years) students who are looking forward to making their bright future in computer science.

Every year we provide participants with numerous and interesting tasks that contribute in open source. According to the number of tasks you do get prizes (certificate, T-Shirt, Hoddie, Grand trip to America). For more information click here.

ABOUT INSTALLATION AND USAGE

Installation and usage of CCEXTRATOR is quite easy.

You can get our downloads for precompiled binaries and source code at our website.

Extracting the subtitles our also quite easy. Just you need to do is run this code:

ccextractor <input>

ccextractor <input>



To get more information of how to use it you can follow these links:

 HYPERLINK "https://www.ccextractor.org/doku.php?id=public:general:command_line_usage" Using the command line tool

 HYPERLINK "https://www.ccextractor.org/doku.php?id=public:general:win_gui_usage" Using the Windows GUI









You can also find the list of parameters and their brief description by running      ccextractor         

 without any arguments.

HOW TO COMPILE CCEXTRACTOR

If want to get more information about compiling build CCEXTRACTOR of your platform just visit this link:

  Compilation guide

  

SUPPORT

If you have any problem, while using or installing CCXTRACTOR we would be happy to solve it. The best way to get our support is by opening an issue at our  HYPERLINK "https://github.com/CCExtractor/ccextractor/issues" issue tracker.

While you are creating a new Issue please fill all the details given in the template provided. This will make us help you as efficiently and fast as we can.

Further, if you have a question or a problem you can also  HYPERLINK "https://www.ccextractor.org/doku.php?id=public:general:support" contact us by email or chat with the team in Slack.

If you want to contribute to CCExtractor but can't submit some code patches or issues or video samples, you can also donate to us. This could be very helpful for us.

CONTRIBUTION

We would be very grateful if you contribute any report issues, Forking forking it, modifying the code and making a pull request to the repository. We have some rules, outlined in the  HYPERLINK "https://github.com/CCExtractor/ccextractor/blob/master/.github/CONTRIBUTING.md" contributor's guide.

NEWS AND OTHER INFORMATION

To get the latest news about releases and modifications just click at this  HYPERLINK "https://github.com/CCExtractor/ccextractor/blob/master/docs/CHANGES.TXT" CHANGES.TXT file.

In case if you need to get more information about CCEXTRACTER visit our website:  HYPERLINK "https://www.ccextractor.org/" https://www.ccextractor.org







THANK YOU!






```

Comment:
- I'm unsure why you try to link to an external website for a logo when there's one already present.
- Windows line-endings mess up everything.